src/components/SubmitButton/SubmitButton.spec.tsx

```diff
@@ -0,0 +1,9 @@
+import { render } from '@testing-library/react';
+
+import SubmitButton from './SubmitButton';
+
+describe('Submit Button component', () => {
+  test('render Submit Button component', () => {
```

Comment:
Tem que usar `it` e não `test`.
A primeira palavra imperativa no ingles. No caso seria "renders", ficando "it renders"